### PR TITLE
TELCODOCS-725: Release Note text for new features in 4.11 for NMO and NHC

### PR DESCRIPTION
**Pull request by ogradyp**
https://issues.redhat.com/browse/TELCODOCS-725 Document new features

Applies to OpenShift version : 4.11

Preview: 
1. [Node Maintenance Operator enhancements](http://file.emea.redhat.com/pogrady/TELCODOCS-725/release_notes/ocp-4-11-release-notes.html#ocp-4-11-nodes-node-maintenance-operator)
2. [Technology Preview features](http://file.emea.redhat.com/pogrady/TELCODOCS-725/release_notes/ocp-4-11-release-notes.html#ocp-4-11-technology-preview)

@slintes can you please review/approve.

Thank you.

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -730,6 +730,15 @@ This release introduces the following customizations for the descheduler:
 
 For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler[Evicting pods using the descheduler].
 
+[id="ocp-4-11-nodes-node-maintenance-operator"]
+==== Node Maintenance Operator enhancements
+The Node Maintenance Operator provides the following enhancements:
+
+* Additional feedback, `drainProgress` and `lastUpdate`, is now provided regarding the status of `NodeMaintenance` CR tasks.
+* For clusters with bare-metal nodes, an easier method is now available where you can place a node into maintenance mode, and resume a node from maintenance mode, by using the web console *Actions* control.
```
Comment (slintes) [inline release_notes/ocp-4-11-release-notes.adoc:738]: maybe add somthing like ".. on the Compute -> Nodes site"?

**Comment by slintes** [comment]
lgtm

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (ogradyp) [reply release_notes/ocp-4-11-release-notes.adoc]: Thanks for you feedback @slintes I have updated it as agreed by including web console, and removing the specific actions.

**Review by lpettyjo — COMMENTED** [review]
LGTM!

**Comment by anna-savina** [comment]
lgtm

**Pull request merged**